buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/DistributedPerformanceTest.groovy

```diff
@@ -370,6 +401,7 @@ class DistributedPerformanceTest extends ReportGenerationPerformanceTest {
         client = new RESTClient("$teamCityUrl/httpAuth/app/rest/9.1")
         client.auth.basic(teamCityUsername, teamCityPassword)
         client.headers['Origin'] = teamCityUrl
+        client.headers['Accept'] = ContentType.JSON.toString()
```

Comment:
@wolfs I think setting this globally here should be fine.

Reply:
Does this work for downloading the artifact? The junit xml is `xml`, right? Or what would be the response there? If it works, then everything is good.

Reply:
The junit xml is `xml`, but we download the whole zip as binary so no issue here. Anyway, I'll make sure everything works on CI, including cancelling.